dask_conversions/done/Dask_Land_Change_geomedian.ipynb

```diff
@@ -4,7 +4,39 @@
    "cell_type": "code",
```

Comment:
<p>Would it be possible to run this for the same area and time extents as the original? So I can visually compare the output? I would do it myself once merged - but I won't be able to run it from my cube environment currently. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Will do</p>

Reply:
<p>thanks</p>